core/controllers/suggestion.py

```diff
@@ -476,6 +476,65 @@ def get(self, target_type, suggestion_type):
 class SuggestionListHandler(base.BaseHandler):
     """Handles list operations on suggestions."""
 
+    URL_PATH_ARGS_SCHEMAS = {}
+
+    HANDLER_ARGS_SCHEMAS = {
+        'GET': {
+            'suggestion_type': {
+                'schema': {
+                    'type': 'basestring'
+                },
+                'default_value': None
```

Comment:
Why have all of these `default_value` set `None`

Reply:
From [this part of the wiki](https://github.com/oppia/oppia/wiki/Schemas#default-and-optional-arguments=), I thought that handlers with optional arguments use the `default_value` key mapping to `None`. From the body of this class's get() method, I believed that the query fields were limited to `feconf.ALLOWED_SUGGESTION_QUERY_FIELDS` but there were no required fields. Please let me know if I misunderstood the code.

Reply:
Hmm, I see. Can you verify that none of these is required (I think you can do that in `suggestion_services.query_suggestion`).